src/Apruve/Order.php

```diff
@@ -14,6 +14,7 @@ class Order extends ApruveObject {
 		'tax_cents',
 		'shipping_cents',
 		'expire_at',
+		'po_number',
```

Comment:
I don't think we include the po_number in the hash generation, do we?

Reply:
We do not, no.

Reply:
I pulled it out of the hash, but it still needs to be in the json, yeah?

Reply:
I believe that is correct.

Reply:
Yes, it does.